exercises/practice/bottle-song/.meta/tests.toml

```diff
@@ -9,26 +9,23 @@
 # As user-added comments (using the # character) will be removed when this file
 # is regenerated, comments can be added via a `comment` key.
 
-[5a02fd08-d336-4607-8006-246fe6fa9fb0]
+[d4ccf8fc-01dc-48c0-a201-4fbeb30f2d03]
 description = "verse -> single verse -> first generic verse"
 
-[77299ca6-545e-4217-a9cc-606b342e0187]
+[0f0aded3-472a-4c64-b842-18d4f1f5f030]
 description = "verse -> single verse -> last generic verse"
 
-[102cbca0-b197-40fd-b548-e99609b06428]
+[f61f3c97-131f-459e-b40a-7428f3ed99d9]
 description = "verse -> single verse -> verse with 2 bottles"
 
-[b8ef9fce-960e-4d85-a0c9-980a04ec1972]
+[05eadba9-5dbd-401e-a7e8-d17cc9baa8e0]
 description = "verse -> single verse -> verse with 1 bottle"
 
-[c59d4076-f671-4ee3-baaa-d4966801f90d]
-description = "verse -> single verse -> verse with 0 bottles"
```

Comment:
Should this match a unit test change?

Reply:
I had a quick look at the history of the canonical data and didn't see that test at all. I might have missed it but I don't know why all the uuids changed.

Reply:
Also, the test suite does not contain this test: https://github.com/exercism/awk/blob/main/exercises/practice/bottle-song/test-bottle-song.bats

Reply:
@rabestro do you recall how tests.toml was generated when you created bottle-song? 

The appropriate ways are: `bin/configlet create --practice-exercise $exercise` or `bin/configlet sync --tests -e $exercise -u`

Reply:
@glennj I found in my log this commands:
```
./bin/configlet lint
./bin/configlet fmt -e bottle-song
```

Reply:
@glennj The script for test generation:

https://github.com/rabestro/exercism-solutions-awk/blob/main/bowling/prompt.txt

Currently I have no wrapper in PowerShell or bash for the prompt, so I just use it directly in AI system.